dev/com.ibm.ws.jdbc/bnd.bnd

```diff
@@ -27,7 +27,8 @@ Export-Package: \
     com.ibm.ws.jdbc.timedoperations, \
     com.ibm.ws.rsadapter, \
     com.ibm.ws.rsadapter.jdbc, \
-    com.ibm.ws.rsadapter.impl
+    com.ibm.ws.rsadapter.impl, \
+    com.ibm.websphere.ce.cm
```

Comment:
this package is also listed under the `Private-package` list. I don't think that will cause an immediate issue but should be cleaned up